web/src/SidebarTriggerButton.scss

```diff
@@ -14,19 +12,68 @@
   justify-content: center;
 }
 
+$trigger-button-color: $color-gray-light;
+$trigger-button-hover-color: $color-white;
+$trigger-button-active-color: $color-green;
+$trigger-button-selected-color: $color-gray-dark;
+
 .SidebarTriggerButton.isReady {
-  background-image: url(assets/svg/trigger-button.svg);
+  background-color: $trigger-button-color;
+  -webkit-mask: url(assets/svg/trigger-button.svg) no-repeat 50% 50%;
+  mask: url(assets/svg/trigger-button.svg) no-repeat 50% 50%;
```

Comment:
Ooh, interesting solution for changing the SVG background image color! I love the concision, and I see that `mask` is now more browser compatible than it used to be. ("Back in my day..." 🧓🏼)